lm-composed/etc/seed-license-in-slurm.py

```diff
@@ -0,0 +1,43 @@
+#!/app/lm-agent/.venv/bin/python
+import subprocess
+
+
+LICENSE_NAME = "test_product.test_feature"
+SERVER_TYPE = "flexlm"
+
+
+def main():
+    output = subprocess.run([
+        "scontrol",
```

Comment:
It might be nice to shlex here to split up a single command string for readability.

Reply:
Done!